scripts/webpack/dev.servers.config.js

```diff
@@ -25,8 +26,8 @@ module.exports = {
     __dirname: false
   },
   entry: {
-    web: [DOTENV, path.join(SERVER_ROOT, 'server.ts')],
-    gqlExecutor: [DOTENV, path.join(GQL_ROOT, 'gqlExecutor.ts')]
+    web: [DOTENV, INIT_PUBLIC_PATH, path.join(SERVER_ROOT, 'server.ts')],
```

Comment:
the public path has to get set before imports use it, so set it here because that way the import order of server.ts doesn't matter